### PR TITLE
Update panel on add-on update

### LINKED ISSUE #44: Panel does not update when add-ons are updated
STR:
1. Install an old version of an add-on from AMO.
2. Open the ACR panel, note down the correct version for the add-on.
3. Use the Firefox add-on update feature to find and install a newer version for the add-on.
4. Open the ACR panel again.

Expected results:
The panel should show the updated add-on.

Actual results:
The panel shows the old add-on.

### PR DESCRIPTION
**Pull request by wagnerand**
Fixes #44.

### CODE REVIEW

File: data/reporter.js [hunk 1]
```diff
@@ -125,7 +125,6 @@ self.port.on("app_e10s_enabled", function(appE10sEnabled, addonE10sEnabled) {
 self.port.on("have_addon_reports", initialize);
 self.port.on("reset", function() {
     document.initialized = false;
-    initialize(document.addonReports);
```
Comment (tofumatt) [inline data/reporter.js]: Why is this deleted? Presumably it's needed to actually reload the panel.

Comment (wagnerand) [inline data/reporter.js]: The panel is reloaded as soon as it's being opened the next time. If we `initialize` here, the panel will be initialized with the old data, instead of the new.

Comment (tofumatt) [inline data/reporter.js]: Gotcha!

**Comment by tofumatt** [comment]
r+
